### PR TITLE
JBPM-8084 - Allow to use Kie Server rendered forms in workbench

### PR DESCRIPTION
**Pull request by mswiderski**

### CODE REVIEW

**Comment by jsoltes** [comment]
@mswiderski I am getting compilation issues, when I try to build this branch locally...
```
[ERROR] /home/jsoltes/Git-repos/kiegroup/droolsjbpm-integration/kie-maven-plugin/src/main/java/org/kie/maven/plugin/BuildMojo.java:[147:32] cannot find symbol
[ERROR]   symbol:   method loadProvider(java.lang.ClassLoader)
[ERROR]   location: class org.drools.compiler.compiler.BPMN2ProcessFactory
```

**Comment by mswiderski** [comment]
@jsoltes not sure what it is but not related so you can just build kie-sever-parent module

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

File: kie-server-parent/kie-server-wars/kie-server/src/main/ee7-resources/WEB-INF/web.xml [hunk 1]
```diff
@@ -17,9 +17,10 @@
     <web-resource-collection>
       <web-resource-name>Swagger web resources</web-resource-name>
       <url-pattern>/services/rest/server/swagger.json</url-pattern>
+      <url-pattern>/services/rest/server/files/*</url-pattern>
       <http-method>GET</http-method>
     </web-resource-collection>
-    <!-- No authentication. Swagger resources should be accessible without authentication. -->
+    <!-- No authentication. Swagger resources should be accessible without authentication and files (form rendering). -->
```
Comment (MarianMacik) [inline kie-server-parent/kie-server-wars/kie-server/src/main/ee7-resources/WEB-INF/web.xml]: Maybe should be `Swagger resources and files (form rendering) should be accessible without authentication.`

File: kie-server-parent/kie-server-wars/kie-server/src/main/webc-resources/WEB-INF/web.xml [hunk 2]
```diff
@@ -41,9 +41,10 @@
     <web-resource-collection>
       <web-resource-name>Swagger web resources</web-resource-name>
       <url-pattern>/services/rest/server/swagger.json</url-pattern>
+      <url-pattern>/services/rest/server/files/*</url-pattern>
       <http-method>GET</http-method>
     </web-resource-collection>
-    <!-- No authentication. Swagger resources should be accessible without authentication. -->
+    <!-- No authentication. Swagger resources should be accessible without authentication and files (form rendering). -->
```
Comment (MarianMacik) [inline kie-server-parent/kie-server-wars/kie-server/src/main/webc-resources/WEB-INF/web.xml]: Same here.

**Pull request merged**